reference/catalogue.yaml

```diff
@@ -597,18 +690,54 @@ components:
     Concept:
       title: Concept
       type: object
-      description: 'What are the actual available types here? - Concept, Period, Place'
```

Comment:
Realised I forgot to replace this with something - any thoughts what it should be that are better than "A concept"?

Reply:
The original OWL went for `"A concept that forms part of a controlled vocabulary."` but that might be a bit more library-like that we want here. On the other hand, the inspiration for the word is SKOS, which goes for the rather wonderful:

> Concepts are the units of thought—ideas, meanings, or (categories of) objects and events—which underlie many knowledge organization systems. As such, concepts exist in the mind as abstract entities which are independent of the terms used to label them.

I'd be tempted to go for whatever matches whatever we have for works and then improve both of them at some point to be more descriptive.

Reply:
The SKOS description is nice! Works is currently:
> An individual work such as a text, archive item or picture; or a grouping of individual works (so, for instance, an archive collection counts as a work, as do all the series and individual files within it).  Each work may exist in multiple instances (e.g. copies of the same book).  N.B. this is not synonymous with \"work\" as that is understood in the International Federation of Library Associations and Institutions'' Functional Requirements for Bibliographic Records model (FRBR) but represents something lower down the FRBR hierarchy, namely manifestation. Groups of related items are also included as works because they have similar properties to the individual ones.

Which also needs some, uh, work ("a work is an individual work" 🤔)... I'll go for something SKOS-y